src/jmap-email/CHANGELOG.md

```
@@ -67,5 +194,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Initial release. Extracted from the
 [Messages](https://github.com/suitenumerique/messages) project.
 
+[ietf-mime]: https://datatracker.ietf.org/doc/draft-chen-email-mime-ambiguity-defense/
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "ietf-mime"

(MD053, link-image-reference-definitions)